architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,18 @@ endif::openshift-dedicated,openshift-online[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+[[architecture-projects-projects-at-install]]
+=== Projects provided at installation
+
+{product-title} comes with a number projects by out of the box, the most relevant to users being the `openshift` project:
```

Comment:
remove "by"

Reply:
To avoid stating what is most relevant or important to the user and to avoid some awkward sentence structure, maybe we can just say:

{product-title} comes with a number projects out of the box, including the `openshift` project:

Reply:
@ahardin-rh yes, I wanted to leave this open for the future when more projects are introduced, or at least more projects are added that are relevant. There are some that are going to be relevant in the future, hence the layout.